client/src/states/home/template.html

```diff
@@ -4,71 +4,46 @@
+            </div>
+          </form>
+        </div>
+        <div id="newTasks" class="col s12">
+          <closed-task-item></closed-task-item>
+        </div>
+      </div>
     </div>
     <div class="col s12 m4">
       <lecture-item></lecture-item>
     </div>
   </div>
 </div>
 
-
-<!-- <div class="main">
-    <div class="row">
-        <div class="col s12">
-            <ul class="tabs">
-                <li class="tab col s3"><a href="#test1">Test 1</a></li>
-                <li class="tab col s3"><a class="active" href="#test2">Test 2</a></li>
-                <li class="tab col s3" ><a href="#test3">Test 3</a></li>
-                <li class="tab col s3"><a href="#test4">Test 4</a></li>
-            </ul>
-        </div>
-        <div id="test1" class="col s12">
-            <h3>Test 1</h3>
-
-            <ul class="collapsible" data-collapsible="accordion">
-                <li>
-                    <div class="collapsible-header"><i class="material-icons">filter_drama</i>First</div>
-                    <div class="collapsible-body"><span>Lorem ipsum dolor sit amet.</span></div>
-                </li>
-                <li>
-                    <div class="collapsible-header"><i class="material-icons">place</i>Second</div>
-                    <div class="collapsible-body"><span>Lorem ipsum dolor sit amet.</span></div>
-                </li>
-                <li>
-                    <div class="collapsible-header"><i class="material-icons">whatshot</i>Third</div>
-                    <div class="collapsible-body"><span>Lorem ipsum dolor sit amet.</span></div>
-                </li>
-            </ul>
-        </div>
-        <div id="test2" class="col s12">
-            <h3>Test 2</h3>
-
-            <ul class="collapsible" data-collapsible="accordion">
-                <li>
-                    <div class="collapsible-header"><i class="material-icons">filter_drama</i>First</div>
-                    <div class="collapsible-body"><span>Lorem ipsum dolor sit amet.</span></div>
-                </li>
-                <li>
-                    <div class="collapsible-header"><i class="material-icons">place</i>Second</div>
-                    <div class="collapsible-body"><span>Lorem ipsum dolor sit amet.</span></div>
-                </li>
-                <li>
-                    <div class="collapsible-header"><i class="material-icons">whatshot</i>Third</div>
-                    <div class="collapsible-body"><span>Lorem ipsum dolor sit amet.</span></div>
-                </li>
-            </ul>
-
-        </div>
-        <div id="test3" class="col s12">Test 3</div>
-        <div id="test4" class="col s12">Test 4</div>
-    </div>
-</div> -->
-
 <script>
-    $(document).ready(function(){
-        $('ul.tabs').tabs();
-        $('.collapsible').collapsible();
-    });
+  $(document).ready(function () {
+    $('ul.tabs').tabs();
+    $('.collapsible').collapsible();
+  });
```

Comment:
Why template handles logic? All JS should be placed into corresponding controller